src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
@chick basically these are all things used by peek/poke, etc. in PeekPokeTester  that I had to recreate when I overrode stuff... The dspPeek, etc. in the original DspTester also didn't properly print out Poke signalName, so, just as in VerilatorBackend, I used getDataNames to get the right signalName (tho I need to double check things work). 

Flatten I needed for peeking aggregates @ the individual element level (but point to the right peek). 

etc. etc.